packages/reporter/src/test/test.tsx

```diff
@@ -192,15 +192,13 @@ class Test extends Component<TestProps> {
       )
     }
 
-    if (appState.studioActive) {
-      controls.push(
-        <Tooltip key={`studio-command-${this.props.model}`} placement='right' title='Add Commands to Test' className='cy-tooltip'>
-          <a onClick={this._launchStudio} className='runnable-controls-studio'>
-            <WandIcon />
-          </a>
-        </Tooltip>,
-      )
-    }
+    controls.push(
```

Comment:
Confusingly these are **always** rendered, but hidden by CSS  by default, then un-hidden [here](https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/runnables/runnables.scss#L88) if `experimentalStudio` is enabled.

Not ideal but this is how it was in 9.x - I'm thinking instead of changing too much, I'm just going to re-introduce Studio with as little changes a possible. We can improve the implementation prior to removing the `experimental` flag - the goal right now is just to re-enable it for users as it was before.